packages/aws-cdk-lib/aws-synthetics/lib/runtime.ts

```diff
@@ -221,6 +221,19 @@ export class Runtime {
    */
   public static readonly SYNTHETICS_NODEJS_PUPPETEER_8_0 = new Runtime('syn-nodejs-puppeteer-8.0', RuntimeFamily.NODEJS);
 
+  /**
+   * `syn-nodejs-puppeteer-9.0` includes the following:
+   * - Lambda runtime Node.js 20.x
+   * - Puppeteer-core version 22.12.1
+   * - Chromium version 126.0.6478.126
+   *
+   * New Features:
+   * - **Bug fixes** Bug fix to enable visual monitoring capabilities.
```

Comment:
Nit:

```
* Bug fixes:
* - **Bug fix**: Enable visual monitoring capabilities.
```

Reply:
@msambol Thanks for the review. I copied the content from https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch_Synthetics_Library_nodejs_puppeteer.html#CloudWatch_Synthetics_runtimeversion-nodejs-puppeteer-9.0.

Reply:
right, I'm just saying it shouldn't be under `New Features:`

Reply:
I think this is fine -- its a little weird but we've already established a pattern of `bug fixes` being listed as a bullet under `new features`.